packages/client/src/views/VerifyCertificate/VerifyCertificatePage.tsx

```diff
@@ -165,13 +171,18 @@ const FETCH_RECORD_DETAILS_FOR_VERIFICATION = gql`
           }
         }
         eventLocation {
+          id
           name
           description
           type
           address {
+            type
+            line
+            district
+            state
             city
-            districtName
-            stateName
+            postalCode
+            country
```

Comment:
we seem to be using only district, state, city and country so we should be able to safely remove the others